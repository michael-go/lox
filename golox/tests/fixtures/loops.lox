var i = 0;
while (i < 5) {
    print i;
    i = i + 1;
}
print "done while";

print "fibonacci:";
var a = 0;
var temp;
for (var b = 1; a < 20; b = temp + b) {
  print a;
  temp = a;
  a = b;
}