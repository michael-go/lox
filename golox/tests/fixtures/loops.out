# exit code: 0
# stdout:
0
1
2
3
4
done
